Instructions for upgrading ValidMind (#490)
validmind/documentation: site/developer/model-documentation/install-and-initialize-client-library.qmd

After you have pasted the code snippet into your development source code and run your code, the Python client library will connect and register with ValidMind. You can now use the {{< var vm.developer >}} to document and test your models, and to upload model documentation and test results to the {{< var vm.platform >}}.

## Upgrade ValidMind

After installing ValidMind,[^5] you'll want to periodically make sure you are on the latest version to access any new features and other enhancements: 

> 1. Within your Jupyter Notebook or developer environment terminal, retrieve the information for the currently installed version of ValidMind:

Reviewer: ```suggestion
1. In your Jupyter Notebook or developer environment, retrieve the information for the currently installed version of ValidMind:
```

(Just simplifying the text a bit.)

    ```python
    %pip show validmind
    ```

::: {.column-margin}
**Example output:**
```bash
Name: validmind
Version: 2.5.15
...
```
:::

> 2. If the version returned is less recent than the version indicated in our production open-source code,[^6] run the following in a code cell or within your terminal:

Reviewer: ```suggestion
2. If the version returned is lower than the version indicated in our production open-source code,[^6] run the following command:
```

(Again, just simplifying the text.)
